src/Asset/CssCollectionGrouper.php

```diff
@@ -16,9 +21,11 @@ public function group(array $css_assets) {
     // Alter inline group so that it isn't aggregated with regular files.
     foreach ($css_assets as &$item) {
       if (!empty($item['inline'])) {
-        $item['group'] = $item['group'] - 1;
```

Comment:
I tried to keep the SMACSS grouping here by subtracting 1. Having a constant for all inline means that we can't categorise the style elements (which may be what we want?). Which approach do you think is better?